tfx/components/transform/executor.py

```diff
@@ -308,13 +326,26 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     transformed_example_artifact = artifact_utils.get_single_instance(
         output_dict[TRANSFORMED_EXAMPLES_KEY])
     transformed_example_artifact.split_names = artifact_utils.encode_split_names(
-        artifact.DEFAULT_EXAMPLE_SPLITS)
-    transformed_train_output = artifact_utils.get_split_uri(
-        output_dict[TRANSFORMED_EXAMPLES_KEY], 'train')
-    transformed_eval_output = artifact_utils.get_split_uri(
-        output_dict[TRANSFORMED_EXAMPLES_KEY], 'eval')
+        splits_config.transform_splits)
+
+    analyze = []
+    for split in splits_config.analyze_splits:
+      data_uri = artifact_utils.get_split_uri(input_dict[EXAMPLES_KEY], split)
+      analyze.append(io_utils.all_files_pattern(data_uri))
+
+    transform = []
+    transform_output_paths = []
```

Comment:
we normally use transform_output as transform_graph
here is transformed_examples (materialization)

Reply:
ah i understand, thanks for the catch